village/src/components/SmurfForm.js

```diff
@@ -16,15 +16,25 @@ class SmurfForm extends Component {
   }
 
   addSmurf(event) {
-    event.preventDefault();
     // add code to create the smurf using the api
-    
+    const {name, age, height} = this.state;
+
```

Comment:
Nice job with the destructuring.  However, if you look around this file, where is `this.state` coming from?
`this.state` must either be created or you would have to pass props down to manage state.